### PR TITLE
gRPC metrics: Add APIs to register gRPC client and server views separately.

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/census-instrumentation/opencensus-java/issues/1209.

### CODE REVIEW

File: contrib/grpc_metrics/src/test/java/io/opencensus/contrib/grpc/metrics/RpcViewsTest.java [hunk 1]
```diff
@@ -37,59 +37,92 @@
...
-    for (View view : RpcViews.RPC_INTERVAL_VIEWS_SET) {
-      assertThat(fakeViewManager.isRegistered(view)).isFalse();
-    }
-    for (View view : RpcViews.GRPC_CUMULATIVE_VIEWS_SET) {
-      assertThat(fakeViewManager.isRegistered(view)).isFalse();
-    }
+    verifyRegistration(
+        fakeViewManager,
+        /* isOldRpcCumulativeViewsRegistered= */ true,
+        /* isOldRpcIntervalViewsRegistered= */ false,
```
[the remark this answers is not in the document]
Comment (songy23) [reply contrib/grpc_metrics/src/test/java/io/opencensus/contrib/grpc/metrics/RpcViewsTest.java]: Sounds good, done.

File: contrib/grpc_metrics/src/test/java/io/opencensus/contrib/grpc/metrics/RpcViewsTest.java [hunk 2]
```diff
@@ -37,93 +39,57 @@
...
+                .addAll(RpcViews.RPC_CUMULATIVE_VIEWS_SET)
+                .addAll(RpcViews.RPC_INTERVAL_VIEWS_SET)
+                .build());
   }
 
   @Test
   public void registerAllGrpcViews() {
     FakeViewManager fakeViewManager = new FakeViewManager();
     RpcViews.registerClientGrpcViews(fakeViewManager);
     RpcViews.registerServerGrpcViews(fakeViewManager);
```
Comment (sebright) [inline contrib/grpc_metrics/src/test/java/io/opencensus/contrib/grpc/metrics/RpcViewsTest.java]: It looks like this test is no longer testing RpcViews.registerAllGrpcViews().

Comment (songy23) [reply contrib/grpc_metrics/src/test/java/io/opencensus/contrib/grpc/metrics/RpcViewsTest.java]: Good catch, I added a similar helper method to register both client and server views.

**Comment by songy23** [comment]
Thanks!

**Pull request merged**